packages/terra-dialog/src/Dialog.scss

```diff
@@ -1,13 +1,54 @@
+@import '~terra-mixins';
+
 :local {
   .dialog {
     height: 100%;
     width: 100%;
   }
 
   .dialog-header {
-    background-color: var(--terra-dialog-header-background-color, transparent);
-    border-bottom: var(--terra-dialog-header-bottom-border, none);
+    align-items: center;
+    background-color: var(--terra-dialog-header-background-color, #f4f4f4);
+    border-bottom: var(--terra-dialog-header-bottom-border, 1px solid #dedfe0);
     color: var(--terra-dialog-header-foreground-color, #1c1f21);
+    display: flex;
+    font-size: var(--terra-dialog-header-font-size);
+    height: 3.587rem;
+    justify-content: space-between;
+    padding: var(--terra-dialog-header-padding, 0.7143rem);
+  }
+
+  .dialog-header-title {
+    margin-right: 0.7143rem;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+
+  .dialog-header-close {
+    float: right;
+    text-align: right;
+  }
+
+  .close {
+    background: var(--terra-dialog-close-button-background, 0 0);
```

Comment:
Is this the expected fallback?

Reply:
Working through it with @bjankord and @neilpfeiffer, they seemed fine with it but are welcome to chime in if I'm wrong.

Reply:
Went ahead and changed. I think transparent makes more sense.

https://github.com/cerner/terra-core/pull/1250/commits/f50ec854d7f1b5fc4628ddf822cb46dc696d0d77